rcl/src/rcl/arguments_impl.h

```diff
@@ -39,6 +39,7 @@ typedef struct rcl_arguments_impl_t
 
   /// Parameter override rules parsed from arguments.
   rcl_params_t * parameter_overrides;
+
```

Comment:
nit: Delete unnecessary new line

Reply:
I disagree about it not being necessary. It helps grouping struct members, and I intentionally wanted to keep `parameter_overrides` and `parameter_files` apart.

Reply:
I commented that because that's the only change you're doing to this file in the PR